test/lib/pgBackRestTest/Module/Real/RealAllTest.pm

```diff
@@ -350,8 +348,8 @@ sub run
         #---------------------------------------------------------------------------------------------------------------------------
         # Restart the cluster to check for any errors before continuing since the stop tests will definitely create errors and the
         # logs will to be deleted to avoid causing issues further down the line. This test is not version specific so is run on only
-        # the expect version.
-        if ($bExpectVersion)
+        # one version.
```

Comment:
happened to notice that either the word 'need' is missing or the word 'to' s/b deleted from line 350?

Reply:
Fix in main at 6b98b3534e15a8d277547eb2932c085bc7515eec.